joint_limits:
  left_wheel:
    has_position_limits: true
    min_position: -90.0
    max_position: 90.0
    has_velocity_limits: true
    max_velocity: 50.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 100.0
    min_effort: -100.0
  right_wheel:
    has_position_limits: true
    min_position: -90.0
    max_position: 90.0
    has_velocity_limits: true
    max_velocity: 50.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 100.0
    min_effort: -100.0
